core/amber/src/main/python/core/runnables/test_batch_operator.py

```diff
@@ -0,0 +1,861 @@
+import time
+from threading import Thread
+
+import pandas
+import pyarrow
+import pytest
+from loguru import logger
+
+from core.models import (
+    ControlElement,
+    DataElement,
+    InputDataFrame,
+    OutputDataFrame,
+    EndOfUpstream,
+    InternalQueue,
+    Tuple,
+    Batch
+)
+from core.runnables import DataProcessor
+from core.util import set_one_of
+from proto.edu.uci.ics.amber.engine.architecture.sendsemantics import (
+    OneToOnePartitioning,
+    Partitioning,
+)
+from proto.edu.uci.ics.amber.engine.architecture.worker import (
+    AddPartitioningV2,
+    ControlCommandV2,
+    ControlReturnV2,
+    QueryStatisticsV2,
+    UpdateInputLinkingV2,
+    WorkerExecutionCompletedV2,
+    WorkerState,
+    WorkerStatistics,
+    LinkCompletedV2,
+)
+from proto.edu.uci.ics.amber.engine.common import (
+    ActorVirtualIdentity,
+    ControlInvocationV2,
+    ControlPayloadV2,
+    LayerIdentity,
+    LinkIdentity,
+    ReturnInvocationV2,
+)
+from pytexera.udf.examples.count_batch_operator import CountBatchOperator
+
+
+logger.level("PRINT", no=38)
+
+
+class TestBatchDataProcessor:
```

Comment:
I suggest we do not use. `TestBatchDataProcessor`, instead, can we add the test cases into the `TestDataProcessor`, as batch is just one api for data processing.